streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedStreamsTasks.java

```diff
@@ -531,6 +531,13 @@ public String toString(final String indent) {
         builder.append(super.toString(indent));
         describe(builder, restoring.values(), indent, "Restoring:");
         describe(builder, suspended.values(), indent, "Suspended:");
+        describe(builder, restoringByPartition.values(), indent, "Restoring partitions:");
```

Comment:
If we add these to `toString` here, we should also add `runningByPartition` to `AssignedTasks#toString`